CONTRIBUTING.md

```diff
@@ -45,7 +45,7 @@ This will ensure that you have all the required development modules installed to
 
 ### Writing the example
 
-You start off by creating a new file inside `live-examples\css-examples\`. The name of this file should match the example you are adding. For example, if you are adding examples for [`border-radius`](https://developer.mozilla.org/en-US/docs/Web/CSS/border-radius) you would call the file `border-radius.html`.
+You start off by creating a new file inside subfolder in `live-examples\css-examples\`. The name of this file should match the example you are adding. For example, if you are adding examples for [`border-radius`](https://developer.mozilla.org/en-US/docs/Web/CSS/border-radius) you would call the file `border-radius.html`. On the other hand, the folder name should match the name of CSS Specification where the example belongs to. For example, `border-radius` is defined at "CSS Backgrounds and Borders Level 3", hence the example should be put at "backgrounds-and-borders" folder. This information is available at "[Specifications](https://developer.mozilla.org/en-US/docs/Web/CSS/border-radius#Specifications)" section of the MDN article.
```

Comment:
> You start off by creating a new file inside subfolder in `live-examples\css-examples\`.

inside the subfolder `live-examples\css-examples\`.

> if you are adding examples for [`border-radius`](https://developer.mozilla.org/en-US/docs/Web/CSS/border-radius) you would call the file `border-radius.html`.

you would call this file

> On the other hand, the folder name should match the name of CSS Specification where the example belongs to.

name of the CSS Specification to which the example belongs.

> For example, `border-radius` is defined at "CSS Backgrounds and Borders Level 3", hence the example should be put at "backgrounds-and-borders" folder.

is part of "CSS Backgrounds and Borders Level 3", hence the example should be created in the "backgrounds-and-borders" folder.

> This information is available at "[Specifications](https://developer.mozilla.org/en-US/docs/Web/CSS/border-radius#Specifications)" section of the MDN article.

This information is available in the specifications section of the documentation on MDN. For example, for `border-radius` this can be found [here](https://developer.mozilla.org/en-US/docs/Web/CSS/border-radius#Specifications).